docs/bokeh/source/conf.py

```diff
@@ -26,6 +26,7 @@
 project = "Bokeh"
 
 version = settings.docs_version() or __version__
+release = version
```

Comment:
one of the few times I actually prefer this form:
```suggestion
release = version = settings.docs_version() or __version__
```